developer/src/kmc/test/test-infrastructureMessages.ts

```diff
@@ -36,31 +40,110 @@ describe('InfrastructureMessages', function () {
     assert.instanceOf<Error>(ncb.messages[0].exceptionVar, Error);
     });
 
-/*
   // ERROR_FileDoesNotExist
 
   it('should generate ERROR_FileDoesNotExist if a file does not exist', async function() {
-    await testForMessage(this, ['invalid-keyboards', 'error_file_does_not_exist.kmn'], CompilerMessages.ERROR_FileDoesNotExist);
+    const ncb = new NodeCompilerCallbacks({logLevel: 'silent'});
+    await unitTestEndpoints.build(makePathToFixture('invalid-keyboards', 'Error_File_Does_Not_Exist.kmn'), '', ncb, {});
+    assert.isTrue(ncb.hasMessage(InfrastructureMessages.ERROR_FileDoesNotExist),
+      `ERROR_FileDoesNotExist not generated, instead got: `+JSON.stringify(ncb.messages,null,2));
```

Comment:
```suggestion
      `ERROR_FileDoesNotExist not generated, instead got: ${JSON.stringify(ncb.messages,null,2)}`);
```

or

```suggestion
      'ERROR_FileDoesNotExist not generated, instead got: '+JSON.stringify(ncb.messages,null,2));
```

This is a style note -- optional change only, here and elsewhere where this applies.

The `` ` `` character delimits [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). You may prefer to use `'`.

Reply:
Agreed.